docs/user-guide.md

```diff
@@ -752,6 +753,21 @@ The Spark Operator provides limited support for resource quota enforcement using
 
 If you are running Spark applications in namespaces that are subject to resource quota constraints, consider enabling this feature to avoid driver resource starvation. Quota enforcement can be enabled with the command line arguments `-enable-resource-quota-enforcement=true`. It is recommended to also set `-webhook-fail-on-error=true`.
 
+## Running Multiple Instances Of The Operator Within The Same K8s Cluster
+
+If you need to run multiple instances of the operator within the same k8s cluster. Therefore, you need to make sure that the running instances should not compete for the same custom resources or pods. You can achieve this:
+
+Either:
+* By specifying a different `namespace` flag for each instance of the operator.
+
+Or if you want your operator to watch specific resources that may exist in different namespaces:
+
+* You need to add custom labels on resources by defining for each instance of the operator a different set of labels in `-label-selector-filter (e.g. env=dev,app-type=spark)`.
+* Run different `webhook` instances by specifying different `-webhook-config-name` flag for each deployment of the operator.
+* Specify different `webhook-svc-name` and/or `webhook-svc-namespace` for each instance of the operator. 
+* Edit the job that generates the certificates `webhook-init` by specifying the namespace and the service name of each instance of the operator, `e.g. "/usr/bin/gencerts.sh", "-n", "ns-op1", "-s", "spark-webhook-op1", "-p"`. Where `spark-webhook-op1` should match what you have specified in `webhook-svc-name`.
```

Comment:
It's worth mentioning how this can be achieved when using Helm to install the operator as both the namespace and name are parameterized in the Helm template that calls `gencerts.sh`.

Reply:
Ok, I proposed somthing. We are used to Kustomize to deploy on GKE. I hope what I proposed makes it clearer.